code/__DEFINES/medical.dm

```diff
@@ -0,0 +1,28 @@
+/// Physical statuses
+#define PHYSICAL_ACTIVE "Active"
+#define PHYSICAL_PHYSICALLY_UNFIT "Physically Unfit"
+#define PHYSICAL_UNCONSCIOUS "Unconscious"
+#define PHYSICAL_DECEASED "Deceased"
+#define PHYSICAL_CANCEL "Cancel"
```

Comment:
cruft? you don't reference this anymore and `MENTAL_CANCEL` is gone

Reply:
Good catch, I missed this one when looking over my code.